src/speech_node/speech.py

```diff
@@ -26,54 +26,133 @@ class bcolors:
 
 
 class TTS(object):
-    def __init__(self):
+    """
+    Bridge from TTS service calls and topic messages to Toyota TTS, introducing a buffer to handle all requests in
+    receiving order
+    """
+    def __init__(self, rate):
+        """
+        Constructor
```

Comment:
Here too: nice that there are docstrings everywhere! Even better would be (again for doc generation):
```python
"""
Constructor

:param rate: ROS parameter for spin-rate
"""
```